library/src/doo/karma.clj

```diff
@@ -9,7 +9,22 @@
 ;; ======================================================================
 ;; Karma Clients
 
-(def envs #{:chrome :firefox :safari :opera :ie})
+(defn- karma-plugin-name [name]
+  (str "karma-" name "-launcher"))
+
+(def ^:private envs
```

Comment:
I made this private to give us options in the future as it's an impl detail we don't want users relying on.